app/components/organization-details/index.hbs

```diff
@@ -20,60 +20,28 @@
   </OrganizationNameHeader>
 </div>
 
-<div local-class='tabs-container'>
-  <ul local-class='tabs-root' role='tablist'>
-    <li local-class='tab-root' role='tab' {{on 'click' this.displayNamespaces}}>
-      <LinkTo
-        data-test-namespace-link
-        local-class='{{this.namespaceClass}}'
-        @route='authenticated.organization.namespaces'
-      >
-        <AkIcon @iconName='assignment' local-class='tab-icon' />
-
-        {{t 'namespaces'}}
-
-        {{#if this.me.org.is_admin}}
-          <span local-class='badge' class='margin-left'>
-            {{this.organization.selected.namespacesCount}}
-          </span>
-        {{/if}}
-      </LinkTo>
-    </li>
-
-    {{#if this.me.org.is_admin}}
-      <li role='tab' local-class='tab-root' {{on 'click' this.displayMembers}}>
-        <LinkTo
-          data-test-member-link
-          local-class='{{this.memberClass}}'
-          @route='authenticated.organization.users'
-        >
-          <AkIcon @iconName='people' local-class='tab-icon' />
-
-          {{t 'users'}}
-
-          <span local-class='badge' class='margin-left'>
-            {{this.organization.selected.membersCount}}
-          </span>
-        </LinkTo>
-      </li>
-    {{/if}}
-
-    <li local-class='tab-root' role='tab' {{on 'click' this.displayTeams}}>
-      <LinkTo
-        data-test-team-link
-        local-class='{{this.teamClass}}'
-        @route='authenticated.organization.teams'
-      >
-        <AkIcon @iconName='groups' local-class='tab-icon' />
-
-        {{t 'teams'}}
-
-        {{#if this.me.org.is_admin}}
-          <span local-class='badge' class='margin-left'>
-            {{this.organization.selected.teamsCount}}
-          </span>
-        {{/if}}
-      </LinkTo>
-    </li>
-  </ul>
-</div>
\ No newline at end of file
+<AkTabs as |Akt|>
```

Comment:
can we rename `ak-tabs` to `ak-tab`

Reply:
I don't see anything wrong with AkTabs. That's what it is actually.